assemble/bin/accumulo-cluster

```diff
@@ -208,7 +208,11 @@ function canRunOnHost() {
     exit 1
   fi
 
-  [[ $(isLocalHost "$1") == 0 ]]
+  if isLocalHost "$1"; then
+    return 0
+  else
+    return 1
+  fi
```

Comment:
Can't you just do:

```suggestion
isLocalHost "$1"
```

The return code of the function is the same as the exit code of the last command in the function.

Reply:
Changed in 239d99d